WooCommerce/Classes/Model/Address+Woo.swift

```diff
@@ -85,6 +85,15 @@ extension Address {
     var isEmpty: Bool {
         self == .empty
     }
+
+    /// Generates an Address object from a TaxRate object data
+    ///
+    static func from(taxRate: TaxRate) -> Address {
+        Address.empty.copy(city: taxRate.cities.first ?? "",
+                           state: taxRate.state,
+                           postcode: taxRate.postcodes.first ?? "",
```

Comment:
These are good questions:

> It's always going to be the first element of cities and postcodes? I'm not sure how are we modelling the data in this case, so I'm assuming the most recent update to these properties is inserted and will come up first?

Basically, by setting any city or postcode from the list it will change the taxes, that's why we take the first one. Likewise we could have taken any city or postcode in the list.

> Will cities and postcodes first elements always match? For example if these collections have different lengths (I'm not positive that's possible) then would we have a miss-match between the city and its postcode?

No, they don't match necessarily. But again, as our final goal is to change the taxes, even if the postcodes and cities don't match the tax will be changed. To make it simple for this iteration we left it so.

We're gathering all these decisions and will create a p2 for the record.

Reply:
Thanks for clarifying!